From: Rik van Riel <riel@surriel.com>
Date: Mon, 14 Jan 2008 15:46:24 -0500
Subject: [x86] cpufreq: unknown symbol fixes
Message-id: 20080114154624.26e66d89@bree.surriel.com
O-Subject: [RHEL5.2 PATCH] fix i686 non-xen cpufreq module
Bugzilla: 427368

WARNING: /lib/modules/2.6.18-62.el5/kernel/drivers/cpufreq/cpufreq_ondemand.ko
needs unknown symbol __udivdi3
WARNING: /lib/modules/2.6.18-62.el5/kernel/drivers/cpufreq/cpufreq_ondemand.ko
needs unknown symbol __udivdi3

Fixes bug 427368

Acked-by: Brian Maly <bmaly@redhat.com>

diff --git a/drivers/cpufreq/cpufreq_ondemand.c b/drivers/cpufreq/cpufreq_ondemand.c
index 8da2861..da644aa 100644
--- a/drivers/cpufreq/cpufreq_ondemand.c
+++ b/drivers/cpufreq/cpufreq_ondemand.c
@@ -337,7 +337,7 @@ static int dbs_calc_load(struct cpu_dbs_info_s *this_dbs_info)
 {
 	struct cpufreq_policy *policy;
 	cputime64_t cur_jiffies;
-	cputime64_t total_ticks, idle_ticks;
+	unsigned int total_ticks, idle_ticks;
 	unsigned int j;
 	int load;
 
